scripts/test.sh

```diff
@@ -107,6 +107,12 @@ if [ "$1" = "--contrib" ]; then
   done
 
   popd > /dev/null 2>&1
+elif [ "$1" = "--no-default-features" ]; then
```

Comment:
Let's have a `--core` that emulates the existing `--contrib`. We only want to test the core library here; it's too expensive to rebuild and test all of the examples. To be clear, you should have an array of features (tls, private-cookies), test once without default features, test once with the default features, and then test with each feature in the array separately enabled.

Reply:
Does this look good? c272ea1